typeDefs.js

```diff
@@ -407,11 +405,12 @@ const typeDefs = gql`
     deleteUser(id: ID!): User
     updateUserById(user: UserInput!, id: ID!): User
     updateUserByToken(user: UserInput!): User
-    confirmUser(token: String!): Boolean
+    confirmUser(token: String!): User
     recoverUser(email: String!, language: Int!): Boolean
     switchUserStatus(id: ID!): LogicalResult
     resetPassword(password: String!, token: String!): Boolean
     checkIfTokenIsValid(token: String!): Boolean
+    sendConfirmationLetter(email: String!, language: Int!): Boolean
```

Comment:
Please, rename it to `sendEmailConfirmation`

Reply:
renamed